schemata/media.yaml

```diff
@@ -0,0 +1,1263 @@
+                type: array
+                example: [329,238]
+                items: 
+                  - 
+                    description: "X coordinate"
+                    type: integer
+                    example: 329
+                  - 
+                    description: "Y coordinate"
+                    type: integer
+                    example: 238
+              - 
+                description: "Bottom-right point of the rectangle"
+                type: array
+                example: [539,372]
+                items: 
+                  - 
+                    description: "X coordinate"
+                    type: integer
+                    example: 539
+                  - 
+                    description: "Y coordinate"
+                    type: integer
+                    example: 372
+      negative_space:
+        description: "Extracted negative spaces from the cover image, represented as rectangles around negative spaces and colors extracted from each negative space"
+        type: object
+        properties:
+          rectangles:
+            description: "An array of rectangles, each rectangle represents a negative space"
+            type: array
+            items:
+              -
+                description: "A bounding rectangle represented by its coordinates and the maximum length of characters that fills inside the rectangle"
+                type: object
+                properties:
+                  x:
+                    description: "X coordinate"
+                    type: integer
+                    example: 0
+                  y:
+                    description: "Y coordinate"
+                    type: integer
+                    example: 378
+                  width:
+                    description: "Width of the rectangle"
+                    type: integer
+                    example: 1015
+                  height:
+                    description: "Height of the rectangle"
+                    type: integer
+                    example: 405
+                  text:
+                    description: "Maximum character lengths that fills inside the rectangle, distributed in three classes: large, medium and small"
+                    type: object
+                    properties:
+                      large:
+                        description: "Maximum character length considering a large font size"
+                        type: integer
+                        example: 321
+                      medium:
+                        description: "Maximum character length considering a medium font size"
+                        type: integer
+                        example: 1071
+                      small:
+                        description: "Maximum character length considering a small font size"
+                        type: integer
+                        example: 1803
+              -
+                type: object
```

Comment:
Whats with the 5 times copy of the whole x,y,width,height,text thing? That can't be right!?